### PR TITLE
Correct typo and link to the correct license file

### PR DESCRIPTION
**Pull request by adityam**

### CODE REVIEW

**Comment by ellismg** [comment]
Looks good to me.  Thanks for this @adityam!

File: README.md [hunk 1]
```diff
@@ -66,4 +66,4 @@ There are many .NET projects on GitHub.
 [Introducing .NET Core]: http://blogs.msdn.com/b/dotnet/archive/2014/12/04/introducing-net-core.aspx
 [coreclr blog post]: http://blogs.msdn.com/b/dotnet/archive/2015/02/03/coreclr-is-open-source.aspx
 [corefx]: http://github.com/dotnet/corefx
-[coreclr]: http://github.com/dotnet/coreclr
\ No newline at end of file
+[coreclr]: http://github.com/dotnet/coreclr
```
Comment (MrMetric) [inline README.md]: What changed here? The old line looks identical to the new line

Comment (Zoltu) [inline README.md]: Newline add or removed (I can never remember how to tell which).

Comment (adityam) [inline README.md]: I used Github's edit button to edit the file. It appears that a post-commit hook in github removed a trailing newline.

**Comment by richlander** [comment]
Appreciated, @adityam.

**Pull request merged**